x-pack/plugins/session_view/common/constants.ts

```diff
@@ -26,6 +28,23 @@ export const KIBANA_DATE_FORMAT = 'MMM DD, YYYY @ hh:mm:ss.SSS';
 //    search functionality will instead use a separate ES backend search to avoid this.
 // 3. Fewer round trips to the backend!
 export const PROCESS_EVENTS_PER_PAGE = 1000;
+
+// As an initial approach, we won't be implementing pagination for alerts.
+// Instead we will load this fixed amount of alerts as a maximum for a session.
+// This could cause an edge case, where a noisy rule that alerts on every process event
+// causes a session to only list and highlight up to 1000 alerts, even though there could
+// be far greater than this amount. UX should be added to let the end user know this is
+// happening and to revise their rule to be more specific.
+export const ALERTS_PER_PAGE = 1000;
+
+// when showing the count of alerts in details panel tab, if the number
+// exceeds ALERT_COUNT_THRESHOLD we put a + next to it, e.g  999+
+export const ALERT_COUNT_THRESHOLD = 999;
+
+// react-query caching keys
+export const QUERY_KEY_PROCESS_EVENTS = 'sessionViewProcessEvents';
+export const QUERY_KEY_ALERTS = 'sessionViewAlerts';
+
```

Comment:
duplicate of https://github.com/elastic/kibana/pull/127500/files#diff-3f0eab56b857974f26a9eb0b3987c0c099c5b8b6506f19c766c9e56273eef26d